packages/gatsby/src/utils/source-nodes.ts

```diff
@@ -93,8 +93,8 @@ export default async ({
 }: {
   webhookBody: unknown
   pluginName?: string
-  parentSpan: Span
-  deferNodeMutation: boolean
+  parentSpan?: Span
+  deferNodeMutation?: boolean
```

Comment:
Called that function in the tests so had to fix types. Both things can be optional (api-runner-node is JS and thus not typed yet)